kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -4327,12 +4327,12 @@ messages:
          string = psPlayerDescription;
       }
       
-      if StringContain(psPlayerDescription,"�")
+      if StringContain(psPlayerDescription,"�")
```

Comment:
I don't know why these files show as changed.  I can't even see a change in them.  They appear identical in a search.

Reply:
This is fixed due to removal in commit d2c5198, resolving